lib/cli/src/frameworks/lit/ts/Button.ts

```diff
@@ -0,0 +1,91 @@
+import { LitElement, html, css } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { styleMap } from 'lit/directives/style-map.js';
+
+@customElement('sb-button')
```

Comment:
Can we make the file names match the custom element name.  So instead of `Button.ts` this would be `sb-button.ts`.  This tends to be what I have seen from lit examples and other web component libraries. https://github.com/PolymerLabs/lit-element-starter-ts/tree/master/src

Reply:
💯  true, let's advocate the best practices!